src/Sylius/Bundle/CoreBundle/Provider/DefaultTaxZoneProvider.php

```diff
@@ -20,23 +20,10 @@
 class DefaultTaxZoneProvider implements ZoneProviderInterface
```

Comment:
I think it could be: ``ChannelBasedDefaultTaxZoneProvider`` because the service is not just "zone provider", but it gives you the default zone - when Sylius can't figure it out from the order. ;)